crates/sui-protocol-config/src/lib.rs

```diff
@@ -169,6 +169,7 @@ const MAX_PROTOCOL_VERSION: u64 = 55;
 //             Enable soft bundle on mainnet.
 // Version 55: Enable enums on mainnet.
 //             Rethrow serialization type layout errors instead of converting them.
+// Version 56: Enable bridge on mainnet.
```

Comment:
Not for this PR, but in the PR that "reverse cherry-picks this" back into `main`, it may be worthwhile calling out that this version cannot be used (since it might be confusing if folks check against testnet).

Reply:
done